ui-admin/src/study/participants/enrolleeView/EnrolleeOverview.tsx

```diff
@@ -110,9 +127,9 @@ export default function EnrolleeOverview({ enrollee, studyEnvContext, onUpdate }
   </>
 }
 
-const formatName = (profile: Profile | undefined) => {
-  if (!profile) {
-    return ''
+const formatName = (profile: Profile | undefined): React.ReactNode => {
+  if (!profile || (!profile.givenName && !profile.familyName)) {
+    return <span className="text-muted fst-italic">name not provided</span>
```

Comment:
```suggestion
    return <span className="text-muted fst-italic">(name not provided)</span>
```

very super minor nit but I think it would scan a bit better with parens